From 9068067e3fc13cc7c89c04ec05271d27cb303810 Mon Sep 17 00:00:00 2001
From: chrome-release-bot <chrome-release-bot@chromium.org>
Date: Sat, 27 Jun 2020 00:22:04 +0000
Subject: [PATCH 09/12] Incrementing VERSION to 83.0.4103.121

TBR=kariah@chromium.org

Change-Id: Ia91d140a59328480f389c94ab85715308bfee2ea
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2271260
Reviewed-by: Chrome Release Bot (LUCI) <chrome-official-brancher@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/4103@{#732}
Cr-Branched-From: 8ad47e8d21f6866e4a37f47d83a860d41debf514-refs/heads/master@{#756066}
---
 chrome/VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/VERSION b/chrome/VERSION
index f7310a92cf3..ff22ff08a83 100644
--- a/chrome/VERSION
+++ b/chrome/VERSION
@@ -1,4 +1,4 @@
 MAJOR=83
 MINOR=0
 BUILD=4103
-PATCH=120
+PATCH=121
-- 
2.28.0

